recipes/newrelic/infrastructure/ohi/redis/debian.yml

```diff
@@ -174,18 +172,19 @@ install:
                 # Please check the documentation to get more information about local
                 # versus remote entities:
                 # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
-                remote_monitoring: true
+                REMOTE_MONITORING: true
 
                 # New users should leave this property as 'true', to uniquely identify the monitored entities when using
                 # Unix sockets.
-                use_unix_socket: false
+                USE_UNIX_SOCKET: true
```

Comment:
@alvarocabanas we're not using unix socket in this config (no path configured), and we're using port. Consequently, per our documentation we should leave the USE_UNIX_SOCKET to false?

Reply:
I changed it to false, because without the path doesn't do a thing, as you say, I will comment it tomorrow with the team to modify the message that we have "New users..." because it doesn't do a thing without the path.